### PR TITLE
comma can't be typed by keys sequence without extra pause in Electron apps

### PR DESCRIPTION
**Pull request by udarrr**
Bug is browser.keys('something with c,o,m,m,a,s') type it to input as 'something with commas' without commas but have to 'something with c,o,m,m,a,s' in Electron apps

### CODE REVIEW

File: packages/webdriverio/src/commands/browser/keys.ts [hunk 1]
```diff
@@ -54,7 +54,15 @@ export async function keys (
      * W3C way of handle it key actions
      */
     const keyAction = this.action('key')
-    keySequence.forEach((value) => keyAction.down(value))
+    keySequence.forEach((value) => {
+        if (!this.isIOS && value === ','){
```
Comment (christian-bromann) [inline packages/webdriverio/src/commands/browser/keys.ts:58]: How is this related to iOS? Should we check for Safari in general?

**Review by christian-bromann — COMMENTED** [review]
Let's add an e2e test for this, maybe somewhere here: `e2e/wdio/headless/test.e2e.ts`

File: packages/webdriverio/src/commands/browser/keys.ts [hunk 2]
```diff
@@ -54,7 +54,15 @@ export async function keys (
...
+            /**
+             * comma can't be typed within sequence without extra pause here more than 200ms
+             */
+            keyAction.pause(225)
```
Comment (christian-bromann) [inline packages/webdriverio/src/commands/browser/keys.ts:62]: Can we make this a `const` and move the variable declaration to the top of the file?

**Review by christian-bromann — COMMENTED** [review]
@wswebcreation what do you think?

**Comment by udarrr** [comment]
> Let's add an e2e test for this, maybe somewhere here: `e2e/wdio/headless/test.e2e.ts`

@christian-bromann it seems the bug relevant only for electron apps, i wasn't able reproduce it in browsers. Any ideas how should it be determined only for electron ?
